deployments/deploy-ssh.sh

```diff
@@ -415,6 +419,7 @@ spec:
             "--storage-upgrade-endpoint-version=$UPGRADE_ENDPOINT_VERSION",
             "--enable-stork-upgrade=$ENABLE_STORK_UPGRADE",
             "--secret-type=$SECRET_TYPE",
+            "--flash-blade-proxy-volumes=$IS_FLASH_BLADE_PROXY_VOLUMES",
```

Comment:
one suggestion, in the future we will support FA proxy volume as well, and our nsm lib is able to arrange request to either FA or FB based on the storage class, so probably a flag like `pure-volumes` might be better?

Reply:
Done